### ISSUE TITLE
Plugin crashed error while running the cypress test with tags

### ISSUE THREAD

**Issue by nimmala89** [bahmutov/cypress-select-tests#142]
I'm facing the below issue while using this plugin.
![image](https://user-images.githubusercontent.com/69274496/90706318-7c62b880-e25a-11ea-8d33-18cda7e10b0d.png)

I have configured the index.js file as given in the documentation.
I have configured my tests to run with a 'smoke' tag

I'm running the command **cypress run --browser chrome --env grep=smoke**

@bahmutov Can you please help me to resolve this ?

**Comment by samlucax** [comment]
Any updates?

**Comment by samlucax** [comment]
I have tested here and I found a scenario: When I use the syntax `const file = require('../../fixtures/example.json')` in my spec file, then the plugins crashes. 

@bahmutov this helps in some way?

**Comment by Ingewar** [comment]
The same thing happened. And right after I'd removed fixtures, everything returns to normal. Any thoughts about what is it and how we can work around it? Cypress version - 4.11.0

**Comment by Ingewar** [comment]
Cypress v5.5.0 - still the same error(

**Comment by Ingewar** [comment]
Apparently, the issue with JSON files. As soon as I convert fixture to .js, everything fires up.

**Comment by samlucax** [comment]
Yeah, I had to do the same. Worked well converting my used JSON files to .js .
